### PR TITLE
[WIP] Playground Phase One

### PR DESCRIPTION
**Pull request by Jougan-0**
**Notes for Reviewers**
Support for anonymous user. This particular PR contains the implementation of anonymous user. When the build is playground mode meshery server creates an anonymous

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: server/models/default_local_provider.go [hunk 1]
```diff
@@ -71,16 +73,177 @@ func (l *DefaultLocalProvider) Initialize() {
...
+		l.PackageURL = providerProperties.PackageURL
+	}
+
+}
+func (l *DefaultLocalProvider) InitializeExtension() {
+	if viper.GetBool("PLAYGROUND") {
+		var extentsion Extensions
+		trueVal := true
+		falseVal := false
+		NavigatorExtension := NavigatorExtension{
```
Comment (MUzairS15) [inline server/models/default_local_provider.go:148]: Remove this.

File: server/models/default_local_provider.go [hunk 2]
```diff
@@ -33,6 +34,7 @@ import (
 	"github.com/spf13/viper"
 	"gopkg.in/yaml.v2"
 	"gorm.io/gorm"
+	"k8s.io/client-go/util/homedir"
```
Comment (MUzairS15) [inline server/models/default_local_provider.go:37]: What is this for?

**Review by MUzairS15 — CHANGES_REQUESTED** [review]
Capabilities will be injected via ENV.

File: server/models/default_local_provider.go (hunk 1 above)
Comment (Jougan-0) [reply server/models/default_local_provider.go:148]: Okay

File: server/models/default_local_provider.go (hunk 2 above)
Comment (Jougan-0) [reply server/models/default_local_provider.go:37]: https://github.com/meshery/meshery/pull/11904/files#diff-0e12afadeea6ac8911508be98ace80904e870a10b2c83c2abe6472b4e6198386R271-R273
For this it was used in the remote provider so I used this.

**Review by MUzairS15 — CHANGES_REQUESTED** [review]
Refactor the function and if/else block.

No need to have Mode as MESHERY.
And please use relatable variable names.

File: server/models/remote_provider.go [hunk 3]
```diff
@@ -81,6 +81,22 @@ const (
 	refURLCookie      = "meshery_ref"
 )
 
+type Name struct {
+	FirstName string `json:"first_name"`
+	LastName  string `json:"last_name"`
+}
+
```
Comment (leecalcote) [inline server/models/remote_provider.go:88]: Rename struct. Add comment.

**Pull request closed without merging**